react/features/whiteboard/reducer.ts

```diff
@@ -0,0 +1,56 @@
+// @ts-ignore
+import { ReducerRegistry } from '../base/redux';
+
+// @ts-ignore
+import { RESET_WHITEBOARD, SETUP_WHITEBOARD } from './actionTypes';
+
+export interface IWhiteboardState {
+
+    /**
+     * The whiteboard collaboration details.
+     */
+    collabDetails: { roomId: string; roomKey: string; } | null;
```

Comment:
Why not flatten this to simplify a bit?

Reply:
I'm eventually passing this as a prop and since this is the expected type in excalidraw, I figured I might as well use the same convention.